gwt-i18n/src/test/java/org/gwtproject/i18n/client/DictionarySuite.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2007 Google Inc.
+ * Copyright 2013 Google Inc.
```

Comment:
why change this, and why change it to 2013?

Reply:
oh, i think git is just being dumb, doesnt understand this file wasnt "moved"

Reply:
On that note, was Localizable.java removed from client? I do think we should keep that one around, since that is a bit more user facing

Reply:
it was deprecated from a very long time ago

```java
/**
 * For backwards compatibility only.
 * 
 * deprecated use {@link org.gwtproject.i18n.shared.Localizable} instead
 */
// Temporarily remove deprecation to keep from breaking teams that don't allow
// deprecated references.
// @Deprecated
public interface Localizable extends org.gwtproject.i18n.shared.Localizable {
}
```